script/validate/vendor

```diff
@@ -18,8 +18,8 @@
 set -eu -o pipefail
 
 if [ -f vendor.conf ]; then
-  rm -rf vendor/
-  vndr |& grep -v -i clone
+  echo "vndr is no longer supported by this check: use go mod vendor instead"
+  exit 1
 elif [ -f go.mod ]; then
```

Comment:
Alternatively, I could just remove this block (I don't think we still have any codebase using `vendor.conf`?)

Reply:
👍 yes that was my train of thought as well; these few lines don't break the bank, and we can remove them at some point